tvaultuiv2/src/views/private/safe/components/SafeDashboard/index.js

```diff
@@ -144,15 +154,74 @@ const SafeDashboard = (props) => {
    */
 
   useEffect(() => {
-    safes.map((item) => {
-      return setSafeList((prev) => [...prev, item]);
-    });
-    setMoreData(true);
-  }, []);
+    async function fetchData() {
+      const safesApiResponse = await apiService.getApiCall(
+        `/vault/v2/ss/sdb/safes`
```

Comment:
All these pathShould be part of apiServices.js

Reply:
fixed